src/Http/Http.Extensions/test/ProblemDetailsDefaultWriterTest.cs

```diff
@@ -503,6 +504,10 @@ public async Task WriteAsync_AddExtensions_WithJsonContext()
         var options = new JsonOptions();
         options.SerializerOptions.TypeInfoResolver = JsonTypeInfoResolver.Combine(CustomProblemDetailsContext.Default, ProblemDetailsJsonContext.Default);
 
+        var mockNamingPolicy = new Mock<JsonNamingPolicy>();
```

Comment:
Instead of modifying the existing tests, can we add a new one?

Reply:
Hi @captainsafia, My local is somehow not working properly and studio is not giving errors. I may need to reinstall devbox.

Since I am traveling somewhere for few days, I would really appreciate if you can help adding dependencies as they are only missing. That can complete the PR and issue. Please let me know. :)